src/Sylius/Bundle/AdminBundle/Menu/MainMenuBuilder.php

```diff
@@ -392,4 +393,26 @@ private function addOfficialSupportSubMenu(ItemInterface $menu): void
             ->setLabelAttribute('icon', 'tabler:certificate')
         ;
     }
+
+ private function addAdministrationSubMenu(ItemInterface $menu): void
+ {
+     $Administration = $menu
+         ->addChild('administration')
+         ->setLabel('sylius.ui.administration')
+         ->setLabelAttribute('icon', 'tabler:lock')
+     ;
+
+     $syliusPlus = $Administration
+         ->addChild('roles')
+         ->setUri('https://sylius.com/plus/')
+         ->setLinkAttribute('target', '_blank')
+         ->setLabel('sylius.ui.roles')
+         ->setLabelAttribute('icon', 'tabler:plus')
+     ;
+
+     $syliusPlus
+         ->setExtra('plus_logo', true)
+     ;
```

Comment:
```suggestion
         ->setLabelAttribute('icon', 'tabler:plus')
         ->setExtra('plus_logo', true)
     ;
```

Can we move it here?